release/long_running_tests/long_running_tests.yaml

```diff
@@ -80,6 +80,19 @@
       timeout: 3600
 
 
+- name: many_logs
+  team: core
+  cluster:
+    app_config: app_config.yaml
+    compute_template: tpl_cpu_1.yaml
```

Comment:
How about using a cluster with small nodes, like 8 CPU, so the memory will be low?

Reply:
It seems like m5.2xlarge in tpl_cpu_1.yaml is only with 8 cpus.